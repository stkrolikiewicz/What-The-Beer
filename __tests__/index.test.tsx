import { render, screen } from '@testing-library/react'
import Home from '@/pages/index'

describe('Home', () => {
  it('renders a heading', () => {
    const { container } = render(<Home />)

    const heading = screen.getByRole('heading', {
      name: /Beers/i,
    })

    expect(heading).toBeInTheDocument()

    // expect(container).toMatchSnapshot()
  })
})