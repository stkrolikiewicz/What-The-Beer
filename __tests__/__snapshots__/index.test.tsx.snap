// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <div>
    <h1>
      Welcome to next.js!
    </h1>
  </div>
</div>
`;
